qiskit/algorithms/optimizers/l_bfgs_b.py

```diff
@@ -68,48 +81,30 @@ def __init__(
                 details of every iteration except n-vectors; iprint = 100 print also the
                 changes of active set and final x; iprint > 100 print details of
                 every iteration including x and g.
-            epsilon: Step size used when approx_grad is True, for numerically
+            eps: If jac is approximated, use this value for the step size.
+            epsilon: (DEPRECATED) Step size used when approx_grad is True, for numerically
                 calculating the gradient
+            options: A dictionary of solver options.
+            max_evals_grouped: Max number of default gradient evaluations performed simultaneously.
+            kwargs: additional kwargs for scipy.optimize.minimize.
         """
-        super().__init__()
+        if factr is not None:
+            warnings.warn(
+                "L_BFGS_B.__init__() keyword argument factr is deprecated and replaced with ftol. "
+                "The relationship between the two is ftol = factr * numpy.finfo(float).eps. "
+                "See https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html.",
+                DeprecationWarning,
+                stacklevel=3,
```

Comment:
Shouldn't this be 2? We usually have 2 everywhere else.
```suggestion
                stacklevel=2,
```

Reply:
Oh, I misunderstood. Thank you.